### DISCUSSION TITLE
New AralTool suggestions

### DISCUSSION THREAD

**Discussion by Yivan000** [tudlang/araltools#47] [Ideas]
Suggest new AralTool ideas here, other than SkedMaker!

Ideally, it is something that can aid students in some way. Generic calculators are not allowed.

**Comment by nzozo** [comment]
hi, is there a filter in where 2 subjects must always be taken together,,,, thank u

**Reply by Yivan000** [reply]
Hello! Can you define what "2 subjects must always be taken together" means? By default, all subjects must be taken, unless you've hidden them.
